### ISSUE TITLE
vulnerability id: "CVE-2024-3817" in latest orca-cli version

### ISSUE THREAD

**Issue by AbhishekRatnawat** [orcasecurity/orca-cli#24]
Please fix the following vulnerability coming from latest version of orca-cli-
```
{
       "target": "usr/bin/orca-cli",
      "category": "lang-pkgs",
       "type": "gobinary",
       "vulnerabilities": [
         {
           "vulnerability_id": "CVE-2024-3817",
           "severity": "CRITICAL",
           "pkg_name": "github.com/hashicorp/go-getter",
           "pkg_path": "",
           "installed_version": "v1.7.3",
           "fixed_version": "1.7.4",
           "cvss_v2_score": "",
           "cvss_v3_score": "9.8",
           "status_summary": {
             "priority": "HIGH",
             "status": "FAILED"
           }
         },
 ```

**Comment by lior-orca** [comment]
Thanks 
We are on it.

**Comment by ayala-orca** [comment]
Fixed in the latest CLI version (v1.61.1)

**Comment by AbhishekRatnawat** [comment]
@ayala-orca There are 2 more CVEs but in warning status, which you might wanna handle may be later on-
```

[2024-05-28T09:20:11.316Z]     {
[2024-05-28T09:20:11.316Z]       "target": "usr/bin/orca-cli",
[2024-05-28T09:20:11.316Z]       "category": "lang-pkgs",
[2024-05-28T09:20:11.316Z]       "type": "gobinary",
[2024-05-28T09:20:11.316Z]       "vulnerabilities": [
[2024-05-28T09:20:11.316Z]         {
[2024-05-28T09:20:11.316Z]           "vulnerability_id": "CVE-2024-3727",
[2024-05-28T09:20:11.316Z]           "severity": "HIGH",
[2024-05-28T09:20:11.316Z]           "pkg_name": "github.com/containers/image/v5",
[2024-05-28T09:20:11.316Z]           "pkg_path": "",
[2024-05-28T09:20:11.316Z]           "installed_version": "v5.29.2",
[2024-05-28T09:20:11.316Z]           "fixed_version": "5.30.1",
[2024-05-28T09:20:11.316Z]           "cvss_v2_score": "",
[2024-05-28T09:20:11.316Z]           "cvss_v3_score": "8.3",
[2024-05-28T09:20:11.316Z]           "status_summary": {
[2024-05-28T09:20:11.316Z]             "priority": "INFO",
[2024-05-28T09:20:11.317Z]             "status": "WARNING"
[2024-05-28T09:20:11.317Z]           }
[2024-05-28T09:20:11.317Z]         },
[2024-05-28T09:20:11.317Z]         {
[2024-05-28T09:20:11.317Z]           "vulnerability_id": "CVE-2023-45288",
[2024-05-28T09:20:11.317Z]           "severity": "MEDIUM",
[2024-05-28T09:20:11.317Z]           "pkg_name": "golang.org/x/net",
[2024-05-28T09:20:11.317Z]           "pkg_path": "",
[2024-05-28T09:20:11.317Z]           "installed_version": "v0.20.0",
[2024-05-28T09:20:11.317Z]           "fixed_version": "0.23.0",
[2024-05-28T09:20:11.317Z]           "cvss_v2_score": "",
[2024-05-28T09:20:11.317Z]           "cvss_v3_score": "5.3",
[2024-05-28T09:20:11.317Z]           "status_summary": {
[2024-05-28T09:20:11.317Z]             "priority": "INFO",
[2024-05-28T09:20:11.317Z]             "status": "WARNING"
[2024-05-28T09:20:11.317Z]           }
[2024-05-28T09:20:11.317Z]         }
[2024-05-28T09:20:11.317Z]       ]
```
